biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -120,6 +120,7 @@
       "File_Parsing.ipynb\r\n",
```

Comment:
<p>Should this solution use <code>query</code>  now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>